src/AuditConfiguration.php

```diff
@@ -172,4 +173,25 @@ public function getRevisionIdFieldType()
     {
         return $this->revisionIdFieldType;
     }
+
+    /**
+     * @return array<string, string[]>
+     */
+    final public function getEntityIgnoredProperties(): array
```

Comment:
Maybe we could add the model name as argument 1 in order to allow this method to check directly against a specific combination of model and property.